src/scenes/home/home.js

```diff
@@ -214,6 +215,7 @@ class Home extends Component {
             <Route path="/benefit" component={Benefit} />
             <Route path="/gala" component={Gala} />
             <Route path="/policy" component={Policy} />
+            <Route path="/opCodeCon" component={OpCodeCon} />
```

Comment:
Let's not introduce the usage of camelCasing for a route paths. It's not consistent with the other route naming conventions. Granted, those aren't a model of consistency - they're a mixture of kebab-case and snake_case, but this would add a *third* naming convention and make it even more confusing to anyone else who adds new routes after this.

I'm reading that kebab-case (separated-by-dashes) is a better option than snake_case (separated_by_underscore) for SEO, and I also find it more readable. I don't know that we really care too much about SEO for the individual terms `op`, `code`, and `con`, though. 

So, how about just changing the route to `"/convention"`, so the URL would be `www.operationcode.org/convention` ?